PurchasesCoreSwift/LocalReceiptParsing/ReceiptParser.swift

```diff
@@ -29,10 +29,12 @@ import Foundation
     }
     
     @objc public func receiptHasTransactions(receiptData: Data) -> Bool {
+        Logger.log(level: .info, message: "Parsing local receipt")
         if let receipt = try? parse(from: receiptData) {
             return receipt.inAppPurchases.count > 0
         }
-        // if the receipt can't be parsed, conservatively return true
+
+        Logger.log(level: .warn, message: "\(#file) \(#function) Could not parse receipt, conservatively returning true")
```

Comment:
Oh those #file and #function are cool

Reply:
yeah! I saw them being used on third-party logging solutions, but I didn't want to be too aggressive with them. there's also `#line` for the line number. 
If we find ourselves using them often, we could make them an optional param, as in `Logger.info("something", withCallDetails: true)` or something